fw-child/template/post-meta.php

```diff
@@ -1,5 +1,5 @@
-<div class="news-meta-block p-3 border border-gray-600">
-	<div class="news-meta-item mb-3 mb-lg-4">
+<div class="news-meta-block p-3 border border-gray-600 row">
+	<div class="news-meta-item mb-3 mb-lg-4 col-md-8 col-xl-16">
```

Comment:
There is no margin between the columns of a same line. Maybe once the builder is configured it won't be a problem (don't hesitate if it's the case), but for now, in my dev environment, the columns are glued together:

![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/a3a621be-994e-4b43-b328-907d2ac76644)

Reply:
@xfrenette-crim indeed, with the builder changes it won't be an issue, but in any case, I pushed a couple of code changes to prevent this issue.